maestro-cli/src/main/java/maestro/cli/command/RecordCommand.kt

```diff
@@ -111,87 +123,21 @@ class RecordCommand : Callable<Int> {
 
-            val uploadProgress = ProgressBar(50)
-            System.err.println("Uploading raw files for render...")
-            val id = client.render(screenRecording, frames) { totalBytes, bytesWritten ->
-                uploadProgress.set(bytesWritten.toFloat() / totalBytes)
-            }
-            System.err.println()
-
-            var renderProgress: ProgressBar? = null
-            var status: String? = null
-            var positionInQueue: Int? = null
-            while (true) {
-                val state = client.getRenderState(id)
-
-                // If new position or status, print header
-                if (state.status != status || state.positionInQueue != positionInQueue) {
-                    status = state.status
-                    positionInQueue = state.positionInQueue
-
-                    if (renderProgress != null) {
-                        renderProgress.set(1f)
-                        System.err.println()
-                    }
-
-                    System.err.println()
-
-                    System.err.println("Status : ${styledStatus(state.status)}")
-                    if (state.positionInQueue != null) {
-                        System.err.println("Position In Queue : ${state.positionInQueue}")
-                    }
-                }
-
-                // Add ticks to progress bar
-                if (state.currentTaskProgress != null) {
-                    if (renderProgress == null) renderProgress = ProgressBar(50)
-                    renderProgress.set(state.currentTaskProgress)
-                }
-
-                // Print download url or error and return
-                if (state.downloadUrl != null || state.error != null) {
-                    System.err.println()
-                    if (state.downloadUrl != null) {
-                        System.err.println("@|bold Signed Download URL:|@".render())
-                        System.err.println()
-                        print("@|cyan,bold ${state.downloadUrl}|@".render())
-                        System.err.println()
-                        System.err.println()
-                        System.err.println("Open the link above to download your video. If you're sharing on Twitter be sure to tag us @|bold @mobile__dev|@!".render())
-                    } else {
-                        System.err.println("@|bold Render encountered during rendering:|@".render())
-                        System.err.println(state.error)
-                    }
-                    break
-                }
-
-                Thread.sleep(2000)
-            }
+            val localOutputFile = outputFile ?: path.resolve("maestro-recording.mp4").toFile()
+            val videoRenderer = if (local) LocalVideoRenderer(
+                frameRenderer = SkiaFrameRenderer(),
+                outputFile = localOutputFile,
+                outputFPS = 25,
+                outputWidthPx = 1920,
+                outputHeightPx = 1080,
```

Comment:
Are these the same specs for the video we used to record with remote renderer?

Reply:
Remote recorder was 30 fps but I think 25 if sufficient